### DISCUSSION TITLE
Enhancement: Let the game be able to read new files added to the directory after it was compiled.

### DISCUSSION THREAD

**Discussion by jelleejamm** [FunkinCrew/Funkin#1591] [Ideas]
This enhancement would be helpful for modders who want to add custom songs into the game without replacing others, or maybe redirect an xml to another spritesheet that wasnt in the game originally. Many modding executables have done this but it would be cool to have it in the original game. Either way, it would be helpful.

**Comment by ILuvGemz** [comment]
Why does this sound similar to https://github.com/ninjamuffin99/Funkin/pull/52 for some reason?

**Comment by jelleejamm** [comment]
> 
> 
> Why does this sound similar to #52 for some reason?

Looking at it, they do seem similar.

**Comment by SoffiMeower** [comment]
52 seems kinda forgotten.
